### PR TITLE
[Docs] Clarify that _ccr/info omits parameters from the response when the follower index is paused.

### PR DESCRIPTION
**Pull request by cjcenizal**
This addresses some of the confusion noted in https://github.com/elastic/elasticsearch/issues/54996.

image

Preview:
http://elasticsearch_55961.docs-preview.app.elstc.co/guide/en/elasticsearch/reference/master/ccr-get-follow-info.html

image

### CODE REVIEW

File: docs/reference/ccr/apis/follow/get-follow-info.asciidoc [hunk 1]
```diff
@@ -76,7 +76,8 @@ replication options and whether the follower indices are active or paused.
 
 //Begin parameters
 `parameters`::
-(object) An object that encapsulates {ccr} parameters.
+(object) An object that encapsulates {ccr} parameters. This object will be
```
Comment (lcawl) [inline docs/reference/ccr/apis/follow/get-follow-info.asciidoc:79]: Not a big deal, but I usually put the conditional part of the sentence first so folks can stop reading when it doesn't apply to them.
```suggestion
(object) An object that encapsulates {ccr} parameters. If the follower index's status is `paused`,
```

File: docs/reference/ccr/apis/follow/get-follow-info.asciidoc [hunk 2]
```diff
@@ -76,7 +76,8 @@ replication options and whether the follower indices are active or paused.
...
+omitted if the follower index's `status` is `paused`.
```
Comment (lcawl) [inline docs/reference/ccr/apis/follow/get-follow-info.asciidoc:80]: ...continued:
```suggestion
this object is omitted.
```

File: docs/reference/ccr/apis/follow/get-follow-info.asciidoc [hunk 3]
```diff
@@ -146,7 +147,7 @@ This example retrieves follower info:
 GET /follower_index/_ccr/info
 --------------------------------------------------
 
-The API returns the following results:
+The API returns the following results if the follower index is `active`:
```
Comment (lcawl) [inline docs/reference/ccr/apis/follow/get-follow-info.asciidoc:150]:
```suggestion
If the follower index is `active`, the API returns the following results:
```

File: docs/reference/ccr/apis/follow/get-follow-info.asciidoc [hunk 4]
```diff
@@ -173,3 +174,19 @@ The API returns the following results:
     ]
 }
 --------------------------------------------------
+
+The API returns the following results if the follower index is `paused`:
```
Comment (lcawl) [inline docs/reference/ccr/apis/follow/get-follow-info.asciidoc:178]:
```suggestion
 If the follower index is `paused`, the API returns the following results:
```

File: docs/reference/ccr/apis/follow/get-follow-info.asciidoc [hunk 5]
```diff
@@ -173,3 +174,19 @@ The API returns the following results:
...
+    "follower_indices" : [
+        {
+            "follower_index" : "follower_index",
+            "remote_cluster" : "remote_cluster",
+            "leader_index" : "leader_index",
+            "status" : "paused"
+        }
+    ]
+}
+--------------------------------------------------
```
Comment (lcawl) [inline docs/reference/ccr/apis/follow/get-follow-info.asciidoc:192]: This snippet is failing with "status: expected String [paused] but was String [active]".  I think we need to skip testing on this snippet or run a pause API and rerun the info API before this snippet.

**Review by lcawl — APPROVED** [review]
LGTM, thanks!

**Comment by cjcenizal** [comment]
Thanks for the much-needed assist @lcawl! Once @martijnvg has approved I will merge.

**Review by martijnvg — APPROVED** [review]
LGTM

**Comment by cjcenizal** [comment]
I didn't backport to 7.7.1 due to merge conflicts.

**Pull request merged**